Rx.xcodeproj/project.pbxproj

```diff
@@ -3896,7 +3896,7 @@
 				INFOPLIST_FILE = "$(SRCROOT)/RxRelay/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-				MACH_O_TYPE = staticlib;
+				MACH_O_TYPE = mh_dylib;
```

Comment:
I believe you can safely delete the `MACH_O_TYPE` entries entirely. They'll default to Xcode's recommendation, which is `mh_dylib`.

Reply:
True, but in a way I always prefer explicitness (not the default will change or anything but still I don't see harm in that). The second thing this helps with is the above `sed` script works. Even though it could replace `MACH_O_TYPE =` as well. Don't have a huge preference to this :)

Reply:
Actually removing it will leave no trace of it in the `project.pbxproj` so in a way I think leaving it explicit allows for better communication and also will allow this manual customization in the form of a bash script like the above